### PR TITLE
Adding support to the NEL header to the core httpheaders plugin

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue https://github.com/zero-24/plg_system_httpheader/issues/43#issuecomment-1341606902

### Summary of Changes

Adding support to the NEL header to the core httpheaders plugin
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/NEL

### Testing Instructions

- Verify that you cant add a "NEL" header
- apply the patch
- confirm you can set the NEL header now

### Actual result BEFORE applying this Pull Request

No support for the NEL header

### Expected result AFTER applying this Pull Request

Now we do support the NEL header

### Link to documentations
Please select:
- [x] Documentation link for docs.joomla.org: https://docs.joomla.org/J4.x:Http_Header_Management
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: plugins/system/httpheaders/httpheaders.xml [hunk 1]
```diff
@@ -81,6 +81,7 @@
 							<option value="report-to">Report-To</option>
 							<option value="strict-transport-security">Strict-Transport-Security</option>
 							<option value="x-frame-options">X-Frame-Options</option>
+							<option value="nel">NEL</option>
```
Comment (Quy) [inline plugins/system/httpheaders/httpheaders.xml]: Sort alpha order.

Comment (zero-24) [reply plugins/system/httpheaders/httpheaders.xml]: Done with https://github.com/joomla/joomla-cms/pull/39378/commits/2b592690a0033557b7d786fa0e591c4d4e9b9a6c thanks

**Comment by richard67** [comment]
Shouldn't that be for 4.3-dev as it's a new feature?

**Comment by zero-24** [comment]
> Shouldn't that be for 4.3-dev as it's a new feature?

A "New feature" tbh I dont think it has to go into 4.3-dev but I'm happy to move it around as soon as 4.3-dev is up-to-date with 4.2-dev branch so i can actually move it around :D

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f29682d36470c282f9c1b8cd08ce3556248619a5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39378">issues.joomla.org/tracker/joomla-cms/39378</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on f29682d36470c282f9c1b8cd08ce3556248619a5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39378">issues.joomla.org/tracker/joomla-cms/39378</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39378">issues.joomla.org/tracker/joomla-cms/39378</a>.</sub>

**Comment by fancyFranci** [comment]
Please move that to 4.3-dev again. Maybe it is not a new feature, but it is not fixing wrong behaviour too :)

**Comment by obuisard** [comment]
Thank you for the PR Tobias @zero-24

**Pull request merged**